waku/v2/node/wakunode2.nim

```diff
@@ -289,14 +289,13 @@ proc unsubscribe*(node: WakuNode, request: FilterRequest) {.async, gcsafe.} =
   waku_node_filters.set(node.filters.len.int64)
 
 
-proc publish*(node: WakuNode, topic: Topic, message: WakuMessage,  rlnRelayEnabled: bool = false) {.async, gcsafe.} =
+proc publish*(node: WakuNode, topic: Topic, message: WakuMessage) {.async, gcsafe.} =
```

Comment:
How come we are change the public API here? Is

> When rlnRelayEnabled is true, a zkp will be generated and attached to the message

No longer true? If so, how do we do or plan to do this now?